t/app/controller/waste_kands_bulky.t

```diff
@@ -116,6 +116,7 @@ FixMyStreet::override_config {
     $echo->mock( 'GetServiceUnitsForObject', sub { [] } );
     $echo->mock( 'GetTasks',                 sub { [] } );
     $echo->mock( 'GetEventsForObject',       sub { [] } );
+    $echo->mock( 'CancelReservedSlotsForEvent', sub { [] } );
```

Comment:
Related to above, could check that this mock (and the others) is always called with a non-nil guid value.